core/src/KokkosExp_MDRangePolicy.hpp

```diff
@@ -327,6 +327,16 @@ struct MDRangePolicy : public Kokkos::Impl::PolicyTraits<Properties...> {
     }
     for (int i = rank_start; i != rank_end; i += increment) {
       const index_type length = m_upper[i] - m_lower[i];
+
+      if (m_upper[i] < m_lower[i]) {
+        std::string msg =
+            "Kokkos::MDRangePolicy bounds error: The lower bound (" +
+            std::to_string(m_lower[i]) + ") is greater than its upper bound (" +
+            std::to_string(m_upper[i]) + ") in rank " + std::to_string(i + 1) +
```

Comment:
How does
rank -> dimension
i+1 -> I
sound?

Reply:
Changed error output this.